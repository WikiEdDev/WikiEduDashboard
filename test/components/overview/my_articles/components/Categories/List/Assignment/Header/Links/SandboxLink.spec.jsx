import React from 'react';
import { shallow } from 'enzyme';
import '../../../../../../../../../testHelper';

import SandboxLink from '../../../../../../../../../../app/assets/javascripts/components/overview/my_articles/components/Categories/List/Assignment/Header/Links/SandboxLink';

describe('SandboxLink', () => {
  it('should show the link', () => {
    const component = shallow(<SandboxLink assignment={{ sandboxUrl: 'url' }} />);
    expect(component.find('a').props().href).to.equal('url');
  });

  it('should include a template in the link if the assignment is a new article', () => {
    const component = shallow(<SandboxLink assignment={{ status: 'new_article' }} />);
    expect(component.find('a').props().href).to.include('Template');
  });
});
