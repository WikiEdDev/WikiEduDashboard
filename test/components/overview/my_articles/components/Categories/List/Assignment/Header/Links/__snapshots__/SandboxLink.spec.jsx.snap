// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SandboxLink should include a template in the link if the assignment is a new article 1`] = `ShallowWrapper {}`;

exports[`SandboxLink should show the link 1`] = `ShallowWrapper {}`;
